.travis.yml

```diff
@@ -0,0 +1,14 @@
+sudo: required
+cache:
+  directories:
+    - $HOME/.m2
+language: java
+jdk:
+  - oraclejdk8
+install: true
```

Comment:
I think the problem in Travis report that we really don't do any `install` for artifacts in this project.
No?